src/Gallery/GalleryItemSettings.tsx

```diff
@@ -41,8 +42,19 @@ export const GalleryItemSettings: FC<GalleryItemSettingsProps> = ({ use, selecte
     });
   };
 
+  const formatFileName = (filename: string) => {
+    const filenameSplit = filename.split('.');
+    return (
+      <>
+        <span className="whppt-gallery-settings__title__name">{filenameSplit[0]}</span>
+        <span>{'.'}</span>
+        <span>{filenameSplit[1]}</span>
+      </>
+    );
+  };
+
```

Comment:
Not sure what this is doing. Could it be a placeholder for a future improvement for longer filenames?

Reply:
This is to have the ellipsis on the file name, but not the extension. So instead of having `longfilenam....`, we would have 
`longfi......jpg`. Can remove it though.

Reply:
Makes sense. I think its good for now. We'll add to it later